src/heretic/p_user.c

```diff
@@ -560,6 +560,15 @@ void P_PlayerThink(player_t * player)
         ultimatemsg = false;    // clear out any chat messages.
         BorderTopRefresh = true;
     }
+    
+    // [crispy] Decrement centered message tics
+    player->centerMessageTics--;      // Can go negative
```

Comment:
This will march towards negative infinity. Wouldn't it make more sense to wrap this whole block in another `if player->centerMessageTics > 0` condition?